ingest-app/src/cmr/ingest/api/translation.clj

```diff
@@ -32,14 +36,47 @@
                                  :umm-json
                                  (umm-spec/generate-metadata context umm :umm-json))))
 
+(def xsl-transformer-cache-name
+  "This is the name of the cache to use for XSLT transformer templates. Templates are thread
+  safe but transformer instances are not."
+  :xsl-transformer-templates)
+
+(def ^:private echo10-iso19115-xslt
+  (io/resource "xslt/echo10_to_iso19115.xsl"))
+
+(defn- get-template
```

Comment:
I think we should replace the cache with a compile-time variable. Unless I'm misunderstanding the code, all of these templates are known ahead of time and cannot change while the application is running.

e.g.
```
(def filename->template
  {"filename1" (xslt/read-template "filename1)
   "filename2" ...})
```

Reply:
This is not my code really. It is copied from search-app. I think the reason cache is used rather than compile time variable is because the underlining JAVA code are interfaces, as a result, they can not be resolved in compile time. If we simply use variable, they are not actually executed at compile time, but rather executed every single time the code is evaluated at run time. That's why we used cache to cache the parsed template (I think).

Reply:
I'm ok with using the in-memory cache in that case.